frontend/src/app/components/connection-settings/connection-settings.component.css

```
@@ -178,3 +178,33 @@
 .audit-toggle {
 	margin-top: 4px;
 }
+
+@media (width <= 600px) {
+	.color-theme {
+		display: grid;
+		grid-template-columns: repeat(2, minmax(0, 1fr));
+		grid-gap: 8px;
```

Stylelint (17.12.0)

[error] Expected "grid-gap" to be "gap" (property-no-deprecated)

(property-no-deprecated)